app/views/doc/_accessibility_cy.html.haml

```diff
@@ -128,22 +181,21 @@
     Gwybodaeth dechnegol am hygyrchedd y wefan hon
 
   %p
-    Mae Cofrestrfa Tir EM wedi ymrwymo i wneud ei gwefan yn hygyrch, yn unol â
+    Mae Cofrestrfa Tir EF wedi ymrwymo i wneud ei gwefan yn hygyrch, yn unol â
     Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Chymwysiadau
-    Symudol) (Rhif 2) 2018. Mae&#8217;r wefan hon yn cydymffurfio&#8217;n llwyr â safon AA
-    Canllawiau Hygyrchedd Cynnwys Gwe fersiwn 2.1.
+    Symudol) (Rhif 2) 2018. Mae&#8217;r wefan hon yn yn cydymffurfio ar y cyfan â safon AA
+    Canllawiau Hygyrchedd Cynnwys Gwe fersiwn 2.2, gyda'r eithriadau cyfyngedig a ddisgrifir uchod.
```

Comment:
This apostrophe might need updating to match the others?
![CleanShot 2024-09-24 at 11 19 08](https://github.com/user-attachments/assets/a8c22a29-8dab-48f6-9727-5b6c0f879176)